rasr/feature_scorer.py

```diff
@@ -126,25 +129,54 @@ def __init__(self, prior_mixtures, scale=1.0, priori_scale=0.0, prior_file=None)
         self.config.normalize_mixture_weights = False
 
 
-class TFLabelContextFeatureScorer(FeatureScorer):
+class OnnxFeatureScorer(FeatureScorer):
     def __init__(
         self,
-        fs_tf_config,
-        contextPriorFile,
-        diphonePriorFile,
-        prior_mixtures,
-        prior_scale,
+        *,
+        mixtures: Union[str, tk.Path],
+        model: Union[str, tk.Path],
+        io_map: Dict[str, str],
+        label_log_posterior_scale: float = 1.0,
+        label_prior_scale: float = 0.7,
```

Comment:
I know this was in the Apptek scorer, but actually providing a default here is maybe counter-productive.

Reply:
I agree. But I would still keep the default `label_log_posterior_scale`, messing around with that can be catastrophic, whereas prior scale might change just a little